apps/web/src/shared/index.ts

```diff
@@ -47,9 +47,9 @@ export const pathGenerator = {
 };
 
 export const publicPathGenerator = {
-  all: workspaceId => `/public-workspace/${workspaceId}/all`,
-  trash: workspaceId => `/public-workspace/${workspaceId}/trash`,
-  shared: workspaceId => `/public-workspace/${workspaceId}/shared`,
+  all: workspaceId => `/share/${workspaceId}/all`,
+  trash: workspaceId => `/share/${workspaceId}/trash`,
+  shared: workspaceId => `/share/${workspaceId}/shared`,
```

Comment:
I think we should remove `trash` and `shared` here? because we don't have such that?